content/docs/1.7.0/advanced-resources/system-backup-restore/backup-longhorn-system.md

```diff
@@ -36,8 +36,7 @@ It includes below resources associating with the Longhorn system:
 - ServiceAccounts
 - StorageClasses
 - Volumes
-
-> **Warning:** Longhorn does not backup `BackingImages`. We will improve this part in the future. See [Restore Longhorn System - Prerequisite](../restore-longhorn-system/#prerequisite) for restoring volumes created with the backing image.
+- BackingImages
```

Comment:
This list is sorted alphabetically. Please move it up.

Reply:
ok got it

Reply:
Fixed.